math/lcm_sum.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file
+ * @brief An algorithm to calculate the sum LCM(1,n) + LCM(2,n) + .. + LCM(n,n), where LCM(i,n)
+ * denotes the Least Common Multiple of the integers i and n. For n greater than or equal to 1.
+ * The value of the sum is calculated by formula:
+ * ∑LCM(i, n) = ((∑(d * ETF(d)) + 1) * n) / 2
+ *
+ * @author [Chesta Mittal](https://github.com/chestamittal)
+ */
+
+#include <iostream> // for std::cin std::cout
+#include <cassert>  // for assert
+#include <vector>   // dor std::vector
+
+/**
+ * Function to compute sum of euler totients in sumOfEulerTotient vector
+ * @param num input number
+ * @returns int
```

Comment:
It returns a value of `int`, but what is it?

Reply:
Changed.